scripts/py_matter_idl/matter_idl/backwards_compatibility.py

```diff
@@ -271,6 +278,11 @@ def _check_cluster_list_compatible(self, original: List[Cluster], updated: List[
 
         for original_cluster in original:
             updated_cluster = updated_clusters.get(original_cluster.name)
+
+            if not_stable(updated_cluster.api_maturity) or not_stable(original_cluster.api_maturity):
```

Comment:
This looks great.  I assume we don't have data in the ZAP database on whether specific structs/attributes/commands are stable or not?

Reply:
I think we have a subset, but for expediting things I only added cluster-level checks. We can add more if we ever add provisional or other tags at lower level.

https://github.com/project-chip/zap/blob/master/zcl-builtin/shared/schema/zcl.xsd#L482 shows it in attribute as well, however that is probably insufficient and we need it for structs too (so we know which structs correspond to provisional bits) and so far it seems our major use is cluster.